#! /bin/sh -e
## serve_on_specified_port, experimental patch by Laszlo 'GCS' Boszormenyi
## to address bug number #242713 , but it seems it can be configured as well
## even...
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: update is called from the Debian maintainer scripts and prints
## DP: non-appropriate output.  Change that.

[ -f debian/patches/00patch-opts ] && . debian/patches/00patch-opts
patch_opts="${patch_opts:--f --no-backup-if-mismatch ${2:+-d $2}}"

if [ $# -lt 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
    -patch) patch $patch_opts -p1 < $0;;
    -unpatch) patch $patch_opts -p1 -R < $0;;
    *)
        echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
        exit 1;;
esac

exit 0
@DPATCH@
--- mailman-2.1.4/Mailman/Utils.py.orig	2004-04-12 22:37:31.000000000 +0000
+++ mailman-2.1.4/Mailman//Utils.py	2004-04-12 22:39:52.000000000 +0000
@@ -248,7 +248,7 @@
             count = fullpath.count('/')
         path = ('../' * count) + target
     else:
-        path = web_page_url + target
+        path = baseurl + target
     return path + mm_cfg.CGIEXT
 
 
